### PR TITLE
chore: scheduled link validation

### PR DESCRIPTION
**Pull request by ennru**
* The first commit schedules link validation to run on PRs and once a week.
* The second commit updates a number of links.

### CODE REVIEW

File: akka-docs/src/main/paradox/routing.md [hunk 1]
```diff
@@ -755,13 +755,6 @@ continue to process its messages as normal, eventually processing the `PoisonPil
 cause the routee to stop. After all routees have stopped the router will itself be stopped
 automatically unless it is a dynamic router, e.g. using a resizer.
 
-@@@ note
-
-Brendan W McAdams' excellent blog post [Distributing Akka Workloads - And Shutting Down Afterwards](http://bytes.codes/2013/01/17/Distributing_Akka_Workloads_And_Shutting_Down_After/)
```
Comment (raboof) [inline akka-docs/src/main/paradox/routing.md:760]: We could link to https://web.archive.org/web/20200218184015/http://bytes.codes/2013/01/17/Distributing_Akka_Workloads_And_Shutting_Down_After/ - but perhaps it's not so relevant anymore nowadays.

File: akka-docs/src/main/paradox/typed/cluster-concepts.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ and membership state transitions.
 ### Gossip
 
 The cluster membership used in Akka is based on Amazon's [Dynamo](https://www.allthingsdistributed.com/files/amazon-dynamo-sosp2007.pdf) system and
-particularly the approach taken in Basho's' [Riak](https://riak.com/technology/architecture/) distributed database.
+particularly the approach taken in Basho's' [Riak](http://riak.com/technology/architecture/) distributed database.
```
Comment (raboof) [inline akka-docs/src/main/paradox/typed/cluster-concepts.md:34]: Que?

**Review by raboof — APPROVED** [review]
one mistake(?), otherwise LGTM

File: scripts/link-validator.conf [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  non-https-whitelist = [
+    "http://cidrdb.org/cidr2015/Papers/CIDR15_Paper16.pdf"
+    "http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.41.7628"
+    "http://spray.io"
+    "http://www.bailis.org/blog/doing-redundant-work-to-speed-up-distributed-queries/"
+    "http://www.cs.columbia.edu/~nahum/w6998/papers/sosp87-timing-wheels.pdf"
+    # document not available anymore
+    "http://www.jaist.ac.jp/~defago/files/pdf/IS_RR_2004_010.pdf"
+    # Sigar site has been removed
+    "http://www.hyperic.com/products/sigar"
```
Comment (ennru) [inline scripts/link-validator.conf:64]: We might want to follow up on these. The links are effectively broken.

File: akka-docs/src/main/paradox/typed/cluster-concepts.md (hunk 2 above)
Comment (ennru) [reply akka-docs/src/main/paradox/typed/cluster-concepts.md]: Replaced it with the Wikipedia page link instead, as their cert setup is broken.

File: scripts/link-validator.conf (hunk 3 above)
Comment (johanandren) [reply scripts/link-validator.conf:64]: https://github.com/akka/akka/issues/31089

File: .github/workflows/link-validator.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+name: Link Validator
+
+on:
+  pull_request:
```
Comment (johanandren) [inline .github/workflows/link-validator.yml:4]: Do we really want it on every pull request? That sounds like a lot of link validating even though links are seldom changed/introduced.

File: .github/workflows/link-validator.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  schedule:
+    - cron:  '0 6 * * 1'
```
Comment (johanandren) [inline .github/workflows/link-validator.yml:6]: Not super important but would be nice if we could manually trigger as well (given that they work as I think they do): 
```yaml
schedule:
     - cron:  '0 6 * * 1'
     - workflow_dispatch:
```

**Review by johanandren — APPROVED** [review]

**Pull request merged**